example-external-task-process/build.gradle

```diff
@@ -18,10 +23,19 @@ dependencies {
     implementation(project(":micronaut-camunda-bpm-feature"))
     implementation("io.micronaut:micronaut-runtime")
     implementation("org.camunda.bpm:camunda-engine-rest-jaxrs2:$camundaVersion")
```

Comment:
When using our integrated REST API, do you still need to use this dependency?

Reply:
alternative would be adding the classes as implementation dependency to the feature project

they are not included in this dependency
`implementation("org.camunda.bpm:camunda-engine-rest:$camundaVersion:classes")`
@tobiasschaefer: as maintainer should I add the dependency `org.camunda.bpm:camunda-engine-rest-jaxrs2` to feature project?

Reply:
this solution makes the feature project bigger without need, 
from my point it would make more sense to document this behavior
to keep the feature as small as possible

what is your (@tobiasschaefer , @martisaw ) opinion?

Reply:
I tryed in feature project gradle file:

```
    implementation("org.camunda.bpm:camunda-engine-rest-jaxrs2:$camundaVersion:classes")
```
and this one
```
    implementation("org.camunda.bpm:camunda-engine-rest-jaxrs2:$camundaVersion")
```

both have no effect
example-external-task-process will not compile

always getting this compile error:

```
> Task :example-external-task-process:kaptKotlin
Note: Creating bean classes for 3 type elementsNote: Creating bean classes for 1 type elements/workspace/sources/camunda/micronaut-camunda-bpm/example-external-task-process/src/main/java/info/novatec/micronaut/camunda/externaltask/process/FetchAndLockContextFactory.java:31: error: cannot find symbol
    FetchAndLockContextListener fetchAndLockContextListener() {
    ^
  symbol:   class FetchAndLockContextListener
  location: class FetchAndLockContextFactoryNote: Creating bean classes for 1 type elements/workspace/sources/camunda/micronaut-camunda-bpm/example-external-task-process/src/main/java/info/novatec/micronaut/camunda/externaltask/process/FetchAndLockContextFactory.java:31: error: @Bean method defines a preDestroy method that does not exist or is not public: contextDestroyed
    FetchAndLockContextListener fetchAndLockContextListener() {
                                ^warning: File for type 'info.novatec.micronaut.camunda.externaltask.process.$FetchAndLockContextFactoryDefinition' created in the last round will not be subject to annotation processing.warning: File for type 'info.novatec.micronaut.camunda.externaltask.process.$FetchAndLockContextFactoryDefinitionClass' created in the last round will not be subject to annotation processing.warning: File for type 'info.novatec.micronaut.camunda.externaltask.process.$FetchAndLockContextFactory$FetchAndLockContextListener0Definition' created in the last round will not be subject to annotation processing.warning: File for type 'info.novatec.micronaut.camunda.externaltask.process.$FetchAndLockContextFactory$FetchAndLockContextListener0DefinitionClass' created in the last round will not be subject to annotation processing.[WARN] Incremental annotation processing requested, but support is disabled because the following processors are not incremental: io.micronaut.annotation.processing.TypeElementVisitorProcessor (DYNAMIC), io.micronaut.annotation.processing.AggregatingTypeElementVisitorProcessor (DYNAMIC), io.micronaut.annotation.processing.PackageConfigurationInjectProcessor (DYNAMIC), io.micronaut.annotation.processing.BeanDefinitionInjectProcessor (DYNAMIC), io.micronaut.annotation.processing.ServiceDescriptionProcessor (DYNAMIC).

> Task :example-external-task-process:kaptKotlin FAILED

> Task :micronaut-camunda-bpm-example:compileJava
Note: Creating bean classes for 6 type elements

FAILURE: Build failed with an exception.


```

Reply:
for now I added a comment why it's needed.